<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>277</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>277</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>7680</width>
    <height>4320</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Pokemon Frame Calculator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/resources/logo.png</normaloff>:/resources/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="lbFrameTarget">
        <property name="text">
         <string>Frame Target</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leFrameTarget"/>
      </item>
      <item>
       <widget class="QLabel" name="lbFrameInit">
        <property name="text">
         <string>Initial Frame</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leFrameInit"/>
      </item>
      <item>
       <widget class="QLabel" name="lbFrameHit">
        <property name="text">
         <string>Frame Hit</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leFrameHit"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnCalculate">
        <property name="text">
         <string>Calculate</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbCalibration">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Calibration</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbResultCalibration">
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>0</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
